src/no-response.ts

```diff
@@ -52,6 +55,37 @@ export default class NoResponse {
     }
   }
 
+  async removeLabels(): Promise<void> {
+    core.debug('Starting removeLabels')
+
+    const { optionalFollowUpLabel } = this.config
+    if (!optionalFollowUpLabel) {
+      return
+    }
+    const payload = github.context.payload as IssuesEvent
+    if (payload.action !== 'closed') {
+      return
+    }
+    const owner = payload.repository.owner.login
+    const repo = payload.repository.name
+    const { number } = payload.issue
+    const issue = { owner, repo, issue_number: number }
+
+    // if the issue closed by the issue author, check if optionalFollowUpLabel is present on the issue and then remove it
+    if (payload.action === 'closed' && payload.issue.user.login === payload.sender.login) {
+      const labels = await this.octokit.rest.issues.listLabelsOnIssue(issue)
```

Comment:
Did you consider avoiding the checking part and just calling the `removeLabel`? 🤔 
It seems that it would just [resolve with `404`](https://docs.github.com/en/rest/issues/labels?apiVersion=2022-11-28#remove-a-label-from-an-issue) status if such a label would not exist. 🤔

Reply:
IMO though, it could be a bit better to simply avoid doing the call if the label not exist, especially because now we have two of such labels to remove, and in most of the issue close events it should not be fired at all or there might be more false positives.

Reply:
Sure, that was just an idea, not necessarily a superior one. 👌 
Was just thinking about ways to minimize the number of API calls.
But, as you've mentioned, in mui-x case there will be quite a lot of cases where we'll end up with just one call - check if any of the relevant label exist - and it many cases - it won't. 👍

Reply:
Yep, thank you for the useful suggestion anyways, I'll try to accommodate it somehow.